### PR TITLE
Feature/DL-36

### PR DESCRIPTION
**Pull request by pako535**

### CODE REVIEW

File: forms/book.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                                                  'placeholder':
+                                                      'Table of contents'}))
+
+    language = SelectField('Language',
+                           choices=languages,
+                           validators=[check_language],
+                           render_kw=({
+                               'class': 'custom-select mb-2 mr-sm-2 mb-sm-0',
+                               'id': 'mySelect',
+                               'placeholder': 'Leanguage'}))
```
Comment (wojwarych) [inline forms/book.py]: Typo! :P

Comment (pako535) [reply forms/book.py]: Oops xD

**Comment by kuszcjan** [comment]
This shouldn't go into `release/110618` instead of `master` ?

**Comment by pako535** [comment]
i change this twice times xD

File: views/index.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 import os
 from send_email.emails import send_email
 
+
+
```
Comment (kuszcjan) [inline views/index.py]: yeah, that those 2 blank lines are absolutely necessary

File: forms/book.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    surname_1 = StringField('Surname_1',
+                            validators=[check_author],
+                            render_kw=({'class': 'inputs',
+                                        'placeholder': 'Surname 1'}))
+
+    first_name_2 = StringField('First name 2',
+                               validators=[check_author],
+                               render_kw=({'class': 'inputs',
+                                           'placeholder': 'First Name 2'}))
+    surname_2 = StringField('Surname 2',
```
[the remark this answers is not in the document]
Comment (oskam) [reply forms/book.py]: It was said on daily two days ago that @pako535 has to mock it like that for this sprint. It is caused by some issues with js.

File: tests/conftest.py [hunk 4]
```diff
@@ -126,6 +126,30 @@ def db_book(session):
...
+    # client.post(client.post(url_for('library.login'), data=data))
+    session.add(aa)
+    session.commit()
+
+    yield a
+
+    # if Author.query.filter_by(first_name=a['first_name']
+    # , last_name=a['last_name']).first():
+    #     session.delete(aa)
+    #     session.commit()
```
[the remark this answers is not in the document]
Comment (pako535) [reply tests/conftest.py]: I was said on daily that if i dont have tests, then I have to send it anyway. So this is working version.

File: forms/book.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (pako535) [reply forms/book.py]: True

**Comment by pako535** [comment]
Sorry for stickler fixes, but when I joined branches I did not notice any errors

**Pull request merged**